build/Foundation/dll/Foundation.def

```diff
@@ -6,6 +6,7 @@ LIBRARY Foundation
 
         ; Foundation ObjC classes
         _OBJC_CLASS_NSArray DATA
+        _OBJC_CLASS__NSProxyOrderedSetArray DATA
```

Comment:
yes, when I tried to compile the foundation dll, it started to complain about this class.